src/api/TKBalance.m

```diff
@@ -0,0 +1,17 @@
+//
+//  TKBalance.m
+//  TokenSdk
+//
+//  Created by Larry Hosken on 11/3/17.
+//  Copyright © 2017 Token Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "TokenSdk.h"
+#import "TKBalance.h"
+
+
+@implementation TKBalance : NSObject {
+}
```

Comment:
nit: you dont need the bracket here. It is needed when you have some private properties.